lib/libzfs/libzfs_pool.c

```diff
@@ -2373,6 +2378,8 @@ zpool_scan(zpool_handle_t *zhp, pool_scan_func_t func, pool_scrub_cmd_t cmd)
 				return (zfs_error(hdl, EZFS_SCRUB_PAUSED, msg));
 			else
 				return (zfs_error(hdl, EZFS_SCRUBBING, msg));
+		}else if (ps && ps->pss_func == POOL_SCAN_RESCRUB) {
```

Comment:
Please correct the style nits reported by checkstyle (which you can run at Delphix with `git zfs-make -p`, and you can see the automated run results here: http://build.zfsonlinux.org/builders/Ubuntu%2017.10%20x86_64%20%28STYLE%29/builds/15217/steps/shell_2/logs/stdio)